treon/task.py

```diff
@@ -27,8 +29,10 @@ def run_tests(self):
                 result += console_output
 
             LOG.info(result)
-        except Exception:  # pylint: disable=broad-except
-            LOG.error(self.error_string(traceback.format_exc()))
+        except CellExecutionError as cell_error:
+            LOG.error(self.error_string(cell_error.traceback))
+        except ConversionException:
+            LOG.exception("Notebook execution failed")
```

Comment:
Should it specify which notebook failed?

Reply:
Yes, that's a good idea.